azure-pipelines.yml

```
@@ -627,22 +627,36 @@ jobs:
       TargetPath: $(Build.ArtifactStagingDirectory)
       ArtifactName: rpm-mariner$(tag)
 
-
 # TODO: rpmbuild on Red Hat UBI 8 is slow for unknown reason. Still working with Red Hat to investigate.
-- job: BuildRpmPackageUbi8
-  displayName: Build Rpm Package Red Hat Universal Base Image 8
+- job: BuildRpmPackages
+  displayName: Build Rpm Packages
   # Do not run this job for Pull Requests due to the slowness
   condition: and(succeeded(), in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'Manual', 'Schedule'))
   pool:
     vmImage: 'ubuntu-20.04'
+  strategy:
+    matrix:
+      ubi8:
```

Let's use its original name

```suggestion
      Red Hat Universal Base Image 8:
```